app/javascript/data/supportedLanguages.js

```diff
@@ -0,0 +1,53 @@
+// Every language in languages.yml will be included in our Languages dropdowns.
+// However, they will all be given values of `false` for `sms`, `email`, and `phone`
+// Unless that field is specifically listed as `true` in `supportedLanguages` below
+
+const supportedLanguages = [
```

Comment:
Idea: what if we defined these attributes right in the backend definition?

Reply:
I strongly support this

Reply:
done!